Makefile

```diff
@@ -23,7 +23,7 @@ test-unit:
 .PHONY: test-acceptance
 test-acceptance:
 	docker-compose -f compose/base.yml pull && docker-compose \
-	-p terraform-provider-mezmo-$(BUILD_TAG) \
+	-p terraform-provider-mezmo-$(shell echo $$BUILD_TAG | tr '[:upper:]' '[:lower:]') \
```

Comment:
Fixes this so it'll run with the new docker compose. See: [0](https://mezmo.slack.com/archives/D026CTN5C5N/p1729714808084409)
https://github.com/answerbook/logdna-web/commit/8a37325a6767f8debe9a6f79c93ab1fa148b1589